<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpRoutingDialog</class>
 <widget class="QDialog" name="HelpRoutingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help for route recording</string>
  </property>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLast">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNext">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>290</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLast">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>451</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string>Route recording gives you change to 
record your driwing.

1. Push start icon for searching the 
satellites. Satellite icon starts flickering 
on screen. When satellites have been found 
satellite icon stops flickering and you can 
start driwing. </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStartPic">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="graphics.qrc">:/new/prefix1/Graphics/route_start.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSatellitePic">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="graphics.qrc">:/new/prefix1/Graphics/satellite_vista.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>491</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string>2. When you have driven route push stop 
icon for stop recording and route view opens.

3. Route view draws your route to the screen
 and gives you information about your driving.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStopPic">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="graphics.qrc">:/new/prefix1/Graphics/route_stop.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>491</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string>4. Push save button for sending your route 
to server. You have to be log in for that.

5. Push new button for start 
routing at beginning.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSavePic">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="graphics.qrc">:/new/prefix1/Graphics/route_save.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPic">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="graphics.qrc">:/new/prefix1/Graphics/route_new.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
